kuma/wiki/models.py

```diff
@@ -1375,14 +1377,7 @@ def other_translations(self):
         """
         Return a list of Documents - other translations of this Document
         """
-        if self.parent is None:
-            return list(self.translations.all().order_by('locale'))
-        else:
-            translations = list(self.parent.translations
-                                .exclude(id=self.id)
-                                .order_by('locale'))
-            # The parent doc should be at first
-            return [self.parent] + translations
+        return self.get_other_translations()
```

Comment:
Should ``other_translations`` be deleted instead? If there are other callers, you've made them much less efficient...

Reply:
There are some other callers that can use `get_other_translations()` in order to be more optimized. It makes them much less efficient but changing that can make this PR more complex to review. I can work on that in different PR if you agree.